teuthology/config.py

```diff
@@ -140,6 +140,7 @@ class TeuthologyConfig(YamlConfig):
         'archive_upload_key': None,
         'archive_upload_url': None,
         'automated_scheduling': False,
+        'default_machine_type': 'smithi',
```

Comment:
This should not be required, because other downstream setups has nothing to do with smithi.

Reply:
@kyr I see, so we should drop this all together?

Reply:
if you mean to abort whole PR, then no, because we need to remove machine type as mandatory option from the usage. And to make it working we may ask @djgalloway to update the config file in sepia.

Reply:
Sorry, I did do this but forgot to follow up with you.  Busy week.

```
dgalloway@teuthology:~$ grep default_machine_type /etc/teuthology.yaml /home/teuthology/.teuthology.yaml /home/teuthworker/.teuthology.yaml 
/etc/teuthology.yaml:default_machine_type: smithi
/home/teuthology/.teuthology.yaml:default_machine_type: smithi
/home/teuthworker/.teuthology.yaml:default_machine_type: smithi
```

Reply:
Thanks @kyr @djgalloway works fine now I guess:
```
ideepika@teuthology:~/teuth_devel/teuthology$ virtualenv/bin/teuthology-suite -vv  --suite rados:thrash-erasure-code -c wip-test-merge-log  --suite-repo https://github.com/ideepika/ceph.git --suite-branch without-fix-48609 --subset 930/10000000 -p 60 --filter="k=4-m=2" --dry-run
```
o/p:
```
2021-03-22 08:04:06,237.237 DEBUG:teuthology.suite.run:Base job config:
branch: wip-test-merge-log
kernel:
  kdb: true
  sha1: distro
machine_type: smithi
```